db/cyndi_setup_test.sql

```diff
@@ -0,0 +1,40 @@
+--
```

Comment:
Going to pull these changes into a separate commit, but here's the cyndi setup for testing.